src/metabase/pulse/render/table.clj

```diff
@@ -62,13 +86,31 @@
     (when bar-width
       [:th {:style (style/style (bar-td-style) (bar-th-style) {:width (str bar-width "%")})}])]])
 
+(defn- render-bar
+  [bar-width normalized-zero]
+  (if (< bar-width normalized-zero)
+    (list
+     [:td {:style (style/style (bar-td-style) {:width :99% :border-right "1px solid black" :padding-right 0})}
+      (render-bar-component style/color-red
+                            false
+                            (normalized-score->pixels (- normalized-zero bar-width))
+                            (normalized-score->pixels bar-width))]
+     [:td {:style (style/style (bar-td-style) {:width :99%})}])
+    (list
+     (when-not (zero? normalized-zero)
+       [:td {:style (style/style (bar-td-style) {:width :99% :border-right "1px solid black"})}])
```

Comment:
If the baseline is 0, we only use one column (with all the positive values). If the baseline is non-zero, all values less than it are negative and go in one column, and there's a second column for the positive values.